src/mugs.js

```diff
@@ -649,6 +662,20 @@ define([
             return attrs.nodeset ? [attrs] : [];
         },
 
+        getSetValues: function (mug) {
+            var ret = [];
+
+            if (mug.p.setValue) {
+                ret = [{
+                    value: mug.p.setValue,
+                    event: mug.isInRepeat() ? 'jr-insert' : 'xforms-ready',
```

Comment:
[This](http://dimagi.github.io/xform-spec/#action-events) makes me think that jr-inserts are only valid for that repeats, so maybe we shouldn't care?